detox/src/ios/expectTwo.js

```diff
@@ -800,6 +798,37 @@ class IosExpect {
   }
 }
 
+function _longPressPointAndDuration(arg1, arg2) {
+  let duration = null;
+  let point = null;
+
+  if (typeof arg1 === 'number' || arg1 === undefined) {
+    duration = arg1;
+
+    _assertValidPoint(arg2);
+    point = arg2;
+  } else if (typeof arg1 === 'object' && arg2 === undefined) {
+    _assertValidPoint(arg1);
+    point = arg1;
+  } else {
+    throw new Error('longPress accepts either a duration (number) or a point ({x: number, y: number}) ' +
+        'as its first argument, and optionally a point as its second argument (if the first argument is a duration).');
+  }
+
+  return { duration, point };
+}
+
+function _assertValidPoint(point) {
```

Comment:
You definitely should visit `utils/assertArgument.js` and reuse the logic for both platforms.

Reply:
done